### ISSUE TITLE
Server error with version 2.7.0 and AEM 6.5.5.0

### ISSUE THREAD

**Issue by joestieg** [Netcentric/accesscontroltool#525]
We've installed the latest version (2.7.0) of AC tools on AEM 6.5.5.0 and we get the following internal server error when we navigate to Tools --> Security --> Netcentric AC Tool:

Cannot serve request to /mnt/overlay/netcentric/actool/content/overview.html/actool in biz.netcentric.cq.tools.actool.ui.AcToolTouchUiServlet
Exception:
java.lang.NullPointerException
	at biz.netcentric.cq.tools.actool.ui.AcToolUiService$RequestParameters.getParam(AcToolUiService.java:349)
	at biz.netcentric.cq.tools.actool.ui.AcToolUiService$RequestParameters.fromRequest(AcToolUiService.java:308)
(see attachment for full error log)
[actool_error.txt](https://github.com/Netcentric/accesscontroltool/files/5672995/actool_error.txt)

**Comment by joestieg** [comment]
Had to go back to version 2.5.5 to get overview.html page to load in AEM 6.5.5. Problem appears to originate in version 2.6.0.

**Comment by joestieg** [comment]
We were able to fix this and make it work by just making one line of change in the code and rebuilding it. 

We changed the code as shown below in AcToolUiService. (deleted _result = defaultValue;_)


```
    static String getParam(final HttpServletRequest req, final String name, final String defaultValue) {
            String result = req.getParameter(name);
            if (result == null) {
                result = defaultValue;
                return defaultValue; 
            }
           
            return result.trim();

        }
```

**Comment by ghenzler** [comment]
@joestieg thanks for the suggestions, PR #530 solves it slightly different but should still fix it. Closing this as duplicate of #521

**Comment by ghenzler** [comment]
@joestieg This is fixed in v2.7.1 that is now available
